io/include/pcl/io/pcd_io.h

```diff
@@ -168,6 +168,45 @@ namespace pcl
       int 
       readHeader (const std::string &file_name, pcl::PCLPointCloud2 &cloud, const int offset = 0);
 
+      /** \brief Read the point cloud data (body) from a PCD stream. 
+        *
+        * Reads the cloud points from a text-formatted stream.  For use after
+        * readHeader(), when the resulting data_type == 0.
+        *
+        * \attention This assumes the stream has been seeked to the position
+        * indicated by the data_idx result of readHeader().
+        *
+        * \param[in] stream the stream from which to read the body.
+        * \param[out] cloud the resultant point cloud dataset to be filled.
+        * \param[in] pcd_version the PCD version of the stream (from readHeader()).
+        *
+        * \return
+        *  * < 0 (-1) on error
+        *  * == 0 on success
+        */
+      int
+      readBody_Ascii (std::istream &stream, pcl::PCLPointCloud2 &cloud, int pcd_version);
```

Comment:
There are already functions with names like `generateHeaderASCII` and `writeBinaryCompressed`, so I'd say the most consistent name will be `readBodyASCII`.